packages/ember-runtime/lib/system/native_array.js

```diff
@@ -138,9 +138,10 @@ var A;
 
 if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.Array) {
   NativeArray.apply(Array.prototype);
-  A = function (arr = []) { return arr; };
+  A = function (arr = []) { return arr || []; };
 } else {
   A = function (arr = []) {
+    if (arr === null) { arr = []; }
```

Comment:
In the native version, you rely on truthiness as a check for `arr`, but in this version you look explicitly for `null` instead of `!arr`. Is there a reason for the difference?